podman/api/tar_utils.py

```diff
@@ -26,7 +26,7 @@ def prepare_dockerignore(anchor: str) -> List[str]:
 
 
 def prepare_containerfile(anchor: str, dockerfile: str) -> str:
-    """Ensure that Dockerfile or a proxy Dockerfile is in context_dir.
+    """Ensure that Dockerfile, or a proxy Dockerfile is in context_dir.
```

Comment:
Does it handle Containerfile too?

Reply:
The developer sets the source file name, if a proxy is created it is `.containerfile.<random string>`. I'll change docstring to make that explicate.